google/cloud/bigtable/examples/run_examples_utils.sh

```diff
@@ -221,7 +221,7 @@ function run_all_table_admin_examples {
   run_example ./table_admin_snippets modify-table "${project_id}" "${INSTANCE}" "${TABLE}"
   run_example ./table_admin_snippets wait-for-consistency-check "${project_id}" "${INSTANCE}" "${TABLE}"
   run_example ./table_admin_snippets generate-consistency-token "${project_id}" "${INSTANCE}" "${TABLE}"
-  local token="$(./table_admin_snippets generate-consistency-token ${project_id} ${INSTANCE} ${TABLE} | awk '{print $5}')"
+  local token="$(./table_admin_snippets generate-consistency-token "${project_id}" "${INSTANCE}" "${TABLE}" | awk '{print $5}' | tr -d '\n')"
```

Comment:
It looks like it is probably because google/cloud/bigtable/examples/table_admin_snippets.cc first emits a (useless) empty line (line 286).  Lets get rid of that and see how things go.

Reply:
Did we find any joy here?

Reply:
i did take a look but this is line 286:

https://github.com/googleapis/google-cloud-cpp/blob/8dcf3bded1598df71603b8ff6038a2a09e68a110/google/cloud/bigtable/examples/table_admin_snippets.cc#L286

Did you mean to point me to a different line?

Reply:
That file has a bit of new code in `master`, I am pretty sure @devbww meant this:

https://github.com/googleapis/google-cloud-cpp/blob/8dcf3bded1598df71603b8ff6038a2a09e68a110/google/cloud/bigtable/examples/table_admin_snippets.cc#L439

Reply:
Yeah ... that one.

Reply:
My bad. Fixed now.